tools/vim/hol.src

```diff
@@ -197,6 +204,7 @@ fu! HOLSelect(l,r)
     return
   endif
   call search(a:r,"ce")
+  normal zv
```

Comment:
is this change supposed to be here?

Reply:
I could certainly move it to the `nnoremap` for `<LocalLeader>a` below (line 255), if that's preferable. My thinking was that if any other `HOLSelect` calls end up in the middle of a fold, they should probably do the unfold too.

Reply:
interesting, ok! as-is seems fine then.